src/fsharp/FSharp.Core/string.fs

```diff
@@ -31,12 +31,15 @@ namespace Microsoft.FSharp.Core
 
         [<CompiledName("Map")>]
         let map (mapping: char -> char) (str:string) =
-            if String.IsNullOrEmpty str then
-                String.Empty
+            if String.IsNullOrEmpty str then String.Empty
```

Comment:
Please keep the original indentation style.

Reply:
Resolved in https://github.com/dotnet/fsharp/pull/9470, where it belonged. Sorry for the noise.